test_timeout.py

```diff
@@ -46,7 +46,7 @@ def test_elapsed(more_ms):
     assert elapsed >= more_ms and elapsed <= less_ms
 
 
-@pytest.mark.parametrize("wrong_timeout", (-3.1, -1, 0.0, 0, 1.0, 1e10))
+@pytest.mark.parametrize("wrong_timeout", (-3.1, 0.0, 1.0, 1e10))
```

Comment:
Why don't we have to test negative int anymore?

Reply:
Yes, I understand this but we should still test the situation that someone add this option for an example `-1`. It won't be part of this test function.

Reply:
Please, check the matter in the new commit.